modules/uploading-an-existing-notebook-file-from-a-git-repository-using-code-server.adoc

```diff
@@ -0,0 +1,33 @@
+:_module-type: PROCEDURE
+
+[id='uploading-an-existing-notebook-file-from-a-git-repository-using-code-server_{context}']
+= Uploading an existing notebook file from a Git repository by using code-server
+
+[role='_abstract']
+You can use the code-server user interface to clone a Git repository into your workspace to continue your work or integrate files from an external project.
+
+.Prerequisites
+* A launched and running code-server workbench server.
+* Read access for the Git repository you want to clone.
+
+.Procedure
+. Copy the HTTPS URL for the Git repository.
++
+** On GitHub, click *&#10515; Code* -> *HTTPS* and click the Copy url to clipboard icon.
+** On GitLab, click *Code* and click the Copy URL icon under *Clone with HTTPS*.
+. In your code-server workbench, from the Activity Bar, click the *Source Control* button (image:images/code-server-source-control-button.png[Source Control button]).
+. Click the *Clone Repository* button.
```

Comment:
I don't see that button

<img width="944" alt="image" src="https://github.com/user-attachments/assets/053ba51b-bc06-4a67-95a8-52ecffd3cc67">

Reply:
When I click Initialize Repository, I get

<img width="272" alt="image" src="https://github.com/user-attachments/assets/bed10eca-4f16-4a69-a538-6921a8617d2e">

Reply:
Updated